packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -24,14 +24,20 @@ String? _getUrlScheme(String url) => Uri.tryParse(url)?.scheme;
 bool _isSafariTargetTopScheme(String url) =>
     _safariTargetTopSchemes.contains(_getUrlScheme(url));
 
+
```

Comment:
There's too much whitespace here, please call `dart format` in the dart files you change to apply the dart formatter.